### PR TITLE
Linux Exploitation Tools

### PR DESCRIPTION
**Pull request by rosplk**
Added a new search and dataset for: `Suspicious Linux Discovery Commands`

This search has an updated logic that has a lookup of suspicious bash commands and covers previously added detections for these tools  `(AutoSUID, LinEnum, LinPeas, Linux Exploit Suggesters)`

Will create a new atomic and PR for Mimipenguin specifically if needed

### CODE REVIEW

**Comment by josehelps** [comment]
Hey I see a few detections here that have HUGE issues, might generate lots of false positives, we should have a chat about improving it, before we merge this in.

**Comment by patel-bhavin** [comment]
Updated the description and added a new search ! Removed atomic searches and combined them into one!

**Comment by patel-bhavin** [comment]
image

File: detections/endpoint/suspcious_linux_discovery_commands.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  confidence: 90
+  context:
+  - Source:Endpoint
+  - Stage:Discovery
+  dataset:
+  - https://media.githubusercontent.com/media/splunk/attack_data/master/datasets/attack_techniques/T1059.004/linux_discovery_tools/sysmon_linux.log
+  impact: 90
+  kill_chain_phases:
+  - Exploitation
+  message: Suspcious Linux Discovery Commands detected on $dest$
```
Comment (MHaggis) [inline detections/endpoint/suspcious_linux_discovery_commands.yml]: `Suspicious` misspell there

**Review by MHaggis — DISMISSED** [review]

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**